storage/tests/system.py

```diff
@@ -179,6 +179,44 @@ def test_crud_bucket_with_requester_pays(self):
             # Exercise 'buckets.delete' w/ userProject.
             with_up.delete()
 
+    @unittest.skipUnless(USER_PROJECT, 'USER_PROJECT not set in environment.')
+    def test_bucket_acls_iam_with_requester_pays(self):
+        new_bucket_name = 'acl-w-user-project' + unique_resource_id('-')
+        created = Config.CLIENT.create_bucket(
+            new_bucket_name, requester_pays=True)
+        self.case_buckets_to_delete.append(new_bucket_name)
+
+        with_up = Config.CLIENT.bucket(
+            new_bucket_name, user_project=USER_PROJECT)
+
+        # Exercise bucket ACL w/ userProject
+        acl = with_up.acl
+        acl.reload()
+        acl.all().grant_read()
+        acl.save()
+        self.assertTrue('READER' in acl.all().get_roles())
```

Comment:
[`assertIn`][1] exists

[1]: https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.assertIn

Reply:
I've updated that in #4085.